common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/MavenTestUtils.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.intellij;
+package com.google.cloud.tools.intellij.testing;
```

Comment:
so this can't live in the new module's test src?

Reply:
It could, it just seemed more general.  Do you want me to move it?